### ISSUE TITLE
usage questions/thoughts

### ISSUE THREAD

**Issue by hjbockholt** [trendscenter/pydfnc#1]
It would be create if the interface to this is more or less BIDSin BIDSout.
I have super-happy to help test and review anything that gets put together for this project.
Thanks for initiating it.

**Comment by bbradt** [comment]
Agreed. For now, the plan is just to take in arbitrary numpy arrays, but making it work with Nifti derivatives is a plan, which will want to use a BIDS spec. If you have ideas for a BIDS derivatives template for dFNC, we can implement that here.
